framework/utils/utils.py

```diff
@@ -1045,21 +1045,3 @@ def _access_check(fn):
           return name
   return None
 
-def orderClusterLabels(originalLables):
```

Comment:
this was unused (actually it has been moved to mathUtils in the past but never removed from here)